pkg/network/openshift_sdn.go

```diff
@@ -204,6 +204,13 @@ func isOpenShiftSDNChangeSafe(prev, next *operv1.NetworkSpec) []error {
 			if checkPrevMTU && *next.Migration.MTU.Network.From != *pn.MTU {
 				errs = append(errs, errors.Errorf("invalid Migration.MTU.Network.From(%d) not equal to the currently applied MTU(%d)", *next.Migration.MTU.Network.From, *pn.MTU))
 			}
+
+			if *next.Migration.MTU.Network.To < MinMTUIPv4 || *next.Migration.MTU.Network.To > MaxMTU {
+				errs = append(errs, errors.Errorf("invalid Migration.MTU.Network.To(%d), has to be in range: %d-%d", *next.Migration.MTU.Network.To, MinMTUIPv4, MaxMTU))
```

Comment:
I just saw this now when re-reviewing the changes and having it force fail:
~~~
[akaris@linux network (pull1276)]$ go test -v -run "TestOVNKubernetesIsSafe" .
=== RUN   TestOVNKubernetesIsSafe
2022/01/25 14:28:47 Detected uplink MTU 1500
2022/01/25 14:28:47 Detected uplink MTU 1500
    ovn_kubernetes_test.go:717: 
        Expected
            <*errors.fundamental | 0xc0004e15f0>: {
                msg: "invalid Migration.MTU.Machine.To(65636), has to be in range: 1280-65536",
                stack: [0x12fb10e, 0x1319c5c, 0x50d182, 0x46a381],
            }
        to match error
            <string>: iinvalid Migration.MTU.Machine.To(65636), has to be in range: 1280-65536
--- FAIL: TestOVNKubernetesIsSafe (0.00s)
FAIL
FAIL	github.com/openshift/cluster-network-operator/pkg/network	0.014s
FAIL
~~~

All of these error messages here:
~~~
errs = append(errs, errors.Errorf("invalid Migration.MTU.Network.To(%d), has to be in range: %d-%d", *next.Migration.MTU.Network.To, MinMTUIPv4, MaxMTU))
~~~

Should have `MaxMTU - 1`, correct? 
~~~
errs = append(errs, errors.Errorf("invalid Migration.MTU.Network.To(%d), has to be in range: %d-%d", *next.Migration.MTU.Network.To, MinMTUIPv4, MaxMTU-1))
~~~

And then, we'd get: `invalid Migration.MTU.Machine.To(65636), has to be in range: 1280-65535`

Sorry for not having seen that earlier.